content/soarian/dstu2.md

```diff
@@ -113,6 +122,20 @@ ID                                | Value\[x] Type      | Description
 <%= json(:SOARIAN_CUSTOM_EXTENSION) %>
 
 
+## Pagination
+
+The pagination links are included in the Bundle when a resource returns multiple items. It is important to follow these Link header values instead of constructing your own URLs.
+
+<%= json(:SOARIAN_LINK_EXTENSION) %>
+
+The possible `relation` values are:
+
+|Name     | Description
+|---------|----------------------------------------
+|`self`   | Shows the URL of the current page of results.
+|`next`   | Shows the URL of the immediate next page of results.
```

Comment:
prefer to format tables this way:

```
 Column | Column
--------|-------------
 data   | data
 data   | longer data

* no leading or trailing pipes
* column width set to widest row
* dashes cover entire table width

Reply:
we'll need to note that 'previous' links can return. seems like this was missed for Millennium - I would prefer to address that as well with this change

Reply:
Table formatting has been done.  Soarian does not support 'previous' link and that is why it was not mentioned.

Reply:
Changes are present here. https://github.com/cerner/fhir.cerner.com/pull/232/commits/416d911dabbb8e9fdd2602dbf306ae20ee43e10d